pkg/collect/mysql.go

```diff
@@ -27,6 +27,38 @@ func Mysql(c *Collector, databaseCollector *troubleshootv1beta2.Database) (Colle
 			databaseConnection.IsConnected = true
 			databaseConnection.Version = version
 		}
+
+		rows, err := db.Query("SHOW VARIABLES")
```

Comment:
This is more for replicated - but should this be "optional"?
If so how do we do that?

Reply:
Also @eochoa260 - i think `rows` is an object that we need to defer cleanup on
```
defer rows.Close()
```